030-ConversationalAI/Coach/Solution-3.md

```diff
@@ -0,0 +1,13 @@
+# Challenge 3: Making API calls
+
+[< Previous Challenge](./Solution-2.md) - **[Home](./Readme.md)** - [Next Challenge>](./Solution-4.md)
+
+## Notes & Guidance
+- Relatively straight forward section as long as they're able to get the key for the API. 
+- They'll need to use the entity pulled out from Solution 2 to make the call to the api, they can do this by setting the entity to a variable and calling it in the HTTP call using syntax similar to: ${user.companysymbol}
+- Make sure they are pulling out the **opening** stock price, not just returning the entire json object.
+   - You can index into the returned object using ${dialog.stock.o}
+
+![Sample logic 1](./Images/Ch3-1.JPG)
+![Sample logic 1](./Images/Ch3-2.JPG)
+[Next Solution - Publish and Teams Integration >](./solution-4.md)
```

Comment:
Remove this link, it isn't work anyway and we already have links at the top.

Reply:
I'm contrarian to Gino.  I personally like the extra nav links at the bottom. :)

Reply:
If you keep them, please make sure the text matches the title header of the document. Same as the Table of Contents links.